### PR TITLE
Mandatory libseat

### PR DESCRIPTION
**Pull request by kennylevinsen**
See https://github.com/swaywm/wlroots/pull/2839

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -316,7 +315,3 @@ summary({
 	'man-pages': scdoc.found(),
 }, bool_yn: true)
 
-if not wlroots_features['libseat']
-	warning('The sway binary must be setuid when compiled without (e)logind or libseat')
```
Comment (emersion) [inline meson.build:320]: We loose this warning when libseat is compiled with the builtin backend. I wonder if it's a big deal. Probably not.

**Review by emersion — APPROVED** [review]
LGTM, thanks!

File: meson.build (hunk 1 above)
Comment (kennylevinsen) [reply meson.build:320]: I guess we could probe pkgconfig in the future.

**Comment by martinetd** [comment]
note README also needs updating (still says "* libseat (optional, for (e)logind and seatd support)")... perhaps with a link to https://git.sr.ht/~kennylevinsen/seatd ? although I guess people can probably find it if they try...
(Yes, I'm this lazy!)

**Comment by martinetd** [comment]
ugh, sorry posted that in the wrong repo -- that was meant for wlroots'which had a similar change.

**Comment by emersion** [comment]
Good catch, done in https://github.com/swaywm/wlroots/commit/9f33d8ad39c26c00ef129419ea1c69c6966e5606

**Pull request merged**